### PR TITLE
Oded/spdk perf test for lf

### PR DESCRIPTION
**Pull request by oded-lb**

### CODE REVIEW

File: examples/nvme/perf/perf.c [hunk 1]
```diff
@@ -475,6 +480,21 @@ submit_single_io(struct ns_worker_ctx *ns_ctx)
 	uint64_t		offset_in_ios;
 	int			rc;
 	struct ns_entry		*entry = ns_ctx->entry;
+	uint64_t current_time;
+
+	if (ns_ctx->io_limit) {
+		current_time = spdk_get_ticks();
+		if ((current_time - ns_ctx->io_limit_time) > g_tsc_rate) {
+			ns_ctx->io_limit_count = 0;
```
Comment (shuki-lb) [inline examples/nvme/perf/perf.c]: suggest to send io_limit count to something like io_submited_cnt

File: examples/nvme/perf/perf.c [hunk 2]
```diff
@@ -475,6 +480,21 @@ submit_single_io(struct ns_worker_ctx *ns_ctx)
...
+			ns_ctx->io_limit_time = current_time;
+		}
+
+		ns_ctx->io_limit_count++;
```
Comment (shuki-lb) [inline examples/nvme/perf/perf.c]: suggest to move this counter update after the next branch check , which means increment the counter only if its actually submitting a command / although no overflow is expected its more readable

**Review by shuki-lb — CHANGES_REQUESTED** [review]

File: examples/nvme/perf/perf.c (hunk 1 above)
Comment (oded-lb) [reply examples/nvme/perf/perf.c]: done

File: examples/nvme/perf/perf.c (hunk 2 above)
Comment (oded-lb) [reply examples/nvme/perf/perf.c]: done

**Pull request merged**